### PR TITLE
APEXMALHAR-2162 fix Kafka dependencies in SQL module.

### PR DESCRIPTION
**Pull request by tweise**

### CODE REVIEW

**Comment by tweise** [comment]
test this please

**Comment by tweise** [comment]
@sandeshh please review/merge - this is to use the same Kafka 0.9 version in all tests.

File: sql/pom.xml [hunk 1]
```diff
@@ -152,52 +152,28 @@
...
-      </exclusions>
     </dependency>
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_2.11</artifactId>
-      <version>0.9.0.0</version>
+      <version>0.9.0.1</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
```
Comment (sandeshh) [inline sql/pom.xml:173]: Dependency was added twice.

Comment (tweise) [reply sql/pom.xml:173]: Different classifier.

**Comment by tweise** [comment]
@chaithu14 @sandeshh CI build passes after turning off 0.10 tests, please merge.

**Pull request merged**